OpenEdXMobile/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -217,4 +217,10 @@ public static void renderHtml(@NonNull TextView textView, @NonNull String html)
             viewText.setSpan(spanObj, start, end, flags);
         }
     }
+
+    @ColorRes
+    private static int getAuthorTextColor(@NonNull IAuthorData authorData) {
```

Comment:
The color is applied to the text (as a span) not the view, and thus should be automatically reset when the text is replaced upon recycling.